java/i18n/src/org/netbeans/modules/i18n/wizard/Util.java

```diff
@@ -179,15 +179,9 @@ static boolean wizardEnabled(Node[] activatedNodes) {
                 FileObject primaryFile = dobj.getPrimaryFile();
                 
                 boolean isLocal;
-                try {
-                    isLocal = !primaryFile.isVirtual()
+                isLocal = !primaryFile.isVirtual()
```

Comment:
Please merge new lines 181 + 182

Reply:
fixed..

Reply:
Seems to have lost some where (see other comments)